src/Microsoft.OData.Client/DataServiceContext.cs

```diff
@@ -692,6 +703,11 @@ internal ClientEdmModel Model
             get { return this.model; }
         }
 
+        /// <summary>
+        /// Gets the client model.
+        /// </summary>
+        internal string IsIeee754Compatible { get; set; }

```

Comment:
Could use a bool or a bool? (Nullable<bool>) if you also want to distinguish between the parameter not being set at all and the parameter being set to false.

Reply:
Fixed.